salt/metalk8s/saltmaster/deployed.sls

```diff
@@ -0,0 +1,31 @@
+{% set salt_master_image = 'salt-master' %}
+{% set salt_master_version = '2018.3.3-1' %}
+{% set version = 'dev' %}
+
+Create salt master directories:
+  file.directory:
+    - user: root
+    - group: root
+    - mode: '0700'
+    - makedirs: true
+    - names:
+      - /etc/salt
+      - /var/cache/salt
+      - /var/run/salt
+
+Install salt master manifest:
+  file.managed:
+    - name: /etc/kubernetes/manifests/salt-master-pod.yaml
+    - source: salt://metalk8s/saltmaster/files/salt-master-pod.yaml.j2
+    - template: jinja
+    - user: root
+    - group: root
+    - mode: '0644'
+    - makedirs: false
+    - backup: false
+    - require:
+      - file: Create salt master directories
+    - defaults:
```

Comment:
I prefer have `require`, `watch`, ... at the end of the step even if `defaults` and `context` can content lots of thing